main/rabbit/definitions.json

```diff
@@ -0,0 +1,277 @@
+{
+  "rabbit_version" : "3.6.10",
+  "users" : [
+	{
+	  "name" : "guest",
+	  "password_hash" : "m8PefsW4FcXeHN8M4B8OCAxHdao=",
+	  "hashing_algorithm" : "rabbit_password_hashing_md5",
+	  "tags" : "administrator"
+	}
+  ],
+  "vhosts" : [
+	{
+	  "name" : "/"
+	},
+	{
+	  "name" : "/lidskasila"
+	}
+  ],
+  "permissions" : [
+	{
+	  "user" : "guest",
+	  "vhost" : "/lidskasila",
+	  "configure" : ".*",
+	  "write" : ".*",
+	  "read" : ".*"
+	},
+	{
+	  "user" : "guest",
+	  "vhost" : "/",
+	  "configure" : ".*",
+	  "write" : ".*",
+	  "read" : ".*"
+	}
+  ],
+  "parameters" : [ ],
+  "global_parameters" : [
+	{
+	  "name" : "cluster_name",
+	  "value" : "rabbit@ef4b8b34ac2c"
+	}
+  ],
+  "policies" : [ ],
+  "queues" : [
```

Comment:
tohle je trochu scifi ne :D je to potreba mit takhle predefined? ten command `rabbit:setup-fabric` to nahodi sam krome fronty na subscriptions

si neumim predstavit ze pri vytvoreni novy queue budeme posilat PR sem